schema/cassandra/temporal/versioned/v1.1/cluster_metadata.cql

```diff
@@ -4,6 +4,8 @@ CREATE TABLE cluster_metadata (
   metadata_partition      int,
   data                    blob,
   data_encoding           text,
+  immutable_data          blob,
```

Comment:
Instead of dropping and recreating table, can you use `ALTER` command to add new columns?

Reply:
And `data`/`data_encoding` should have some default values (null?) now again :-(.

Reply:
We should be able to handle this in code by doing dual read/write.
Addressed in the recent revision. Check it out.